rundmc/execrunner/execrunner_windows.go

```diff
@@ -37,16 +34,43 @@ func (e *DirectExecRunner) Run(log lager.Logger, processID string, spec *runrunc
 		return nil, errors.Wrap(err, "marshaling process spec")
 	}
 
-	specPath := filepath.Join(processesPath, proc.ID(), "spec.json")
-	if err := e.FileWriter.WriteFile(specPath, marshaledSpec, 0600); err != nil {
+	processPath := filepath.Join(processesPath, proc.ID())
+	specPath := filepath.Join(processPath, "spec.json")
+
+	if err := os.MkdirAll(processPath, 0700); err != nil {
+		return nil, err
+	}
+
+	if err := ioutil.WriteFile(specPath, marshaledSpec, 0600); err != nil {
 		return nil, errors.Wrap(err, "writing process spec")
 	}
 
-	cmd := exec.Command(e.RuntimePath, "--debug", "--log", "IGNORED", "exec", "-d", "-p", specPath, "--pid-file", "IGNORED", handle)
+	cmd := exec.Command(e.RuntimePath, "--debug", "--log", "IGNORED", "exec", "-p", specPath, "--pid-file", "IGNORED", handle)
+	cmd.Stdout = io.Stdout
+	cmd.Stderr = io.Stderr
 	if err := e.CommandRunner.Start(cmd); err != nil {
 		return nil, errors.Wrap(err, "execing runtime plugin")
 	}
 
+	proc.mux.Lock()
+
```

Comment:
We actually started with that implementation but felt that a RWMutex was clearer since we weren't putting anything on the channel. If using a `chan(struct{})` is more idiomatic than a mutex then we could do that instead.